packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -95,27 +102,30 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
           opacity: 0.5;
         `}
       />
-      <div
+      <EuiSplitPanel.Outer
         css={css`
+          margin: ${euiTheme.size.xs};
+          height: 99%;
           position: absolute;
           top: 0;
           bottom: 0;
           right: 0;
           left: ${left};
           z-index: 1000;
         `}
+        className="eui-yScroll"
+        data-test-subj={PREVIEW_SECTION}
       >
-        <EuiPanel
-          css={css`
-            margin: ${euiTheme.size.xs};
-            height: 100%;
-          `}
-          data-test-subj={PREVIEW_SECTION}
-        >
+        <EuiSplitPanel.Inner grow={false} color="warning" paddingSize="none">
+          <EuiText textAlign="center" color="warning" size="s">
```

Comment:
While I was refactoring this... currently all the params of flyout goes to url, so passing in a component causes url to be `eventFlyout=(preview:(bannerComponent:(_owner:!n,_store:(),key:!n,props:(children:test),ref:!n),...`

Reply:
hmmm I see, not ideal at all... I'm sorry for leading you on a wrong path. I feel like - though it would be nice to have something this flexible - maybe it's not super important at the moment...

Here's what I'm thinking:
- let's keep the optional props renamed to `banner` like you have now
- let's make it an object instead of a string, with 2 properties
```
    title: string;
    color: "transparent", "accent", "primary", "success", "warning", "danger", "subdued", "plain" (like in the [doc](https://elastic.github.io/eui/#/layout/panel#colors-and-corners))
```

That way, devs can decide to not show the banner at all, or show it with different text and background color... not quite as flexible as a fully blown component but probably a good middle ground.

Wdyt?